ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
One compromise I can see is to comment that the connection is not being closed because of the mock framework not requiring it? My point is that without commenting the close is not done it looks wrong to have not closed the connection.